perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -356,7 +360,7 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
   enable_latency_histograms = FLAGS.netperf_enable_histograms or num_streams > 1
   # Throughput benchmarks don't have latency histograms
   enable_latency_histograms = enable_latency_histograms and \
-      benchmark_name != 'TCP_STREAM'
+      (benchmark_name != 'TCP_STREAM' and benchmark_name != 'UDP_STREAM')
```

Comment:
More readable as
(benchmark_name not in ['TCP_STREAM' , 'UDP_STREAM'])

Also if you could fix the \ continue as well, that'd be great.

Reply:
when testing UDP_STREAM, i found the default send socket buffer size to be 212,992 bytes, which is larger than the max possible UDP message size of 65,507 bytes, so it should not be a problem and we don't need the -s flag. I can add in an upper limit of 65507 bytes, because it will cause an error if you try to use a larger size than that.

Reply:
What platforms have you tested this on?  My reading is that there may be some where `-s` is required (no idea which) but I don't see a downside to specifying both `-s` and `-S`

Reply:
I've test on ubuntu18, debian9, and centos8. So far I haven't needed to specify the -s/-S flags. But I can certainly certainly add them in as flags and just give them the default values netperf has given them so far in my testing.